web/src/stream_data.js

```diff
@@ -846,3 +846,25 @@ export function initialize(params) {
 export function remove_default_stream(stream_id) {
     default_stream_ids.delete(stream_id);
 }
+
+export function options_for_dropdown_widget() {
```

Comment:
Should there be a verb at the beginning of this function, like `get_` or `build_` or `prepare_`?

Reply:
Yeah, used `get_`. :)